### DISCUSSION TITLE
Checkmark Widget colors for active streaks

### DISCUSSION THREAD

**Discussion by Waog** [iSoron/uhabits#829] [Feature Requests]
Hi, thx for the great app 🙏☺️

The Checkmark Widget currently has three states:
![Screenshot_20210305-012411](https://user-images.githubusercontent.com/5098472/110049140-8a3cde80-7d51-11eb-8182-26300dbca5b7.png)

It would be nice if the background color of the _active streak_ version would differ more from the _unchecked_ version.

A similar background color to the apps calendar would be helpful:

![Screenshot_20210305-012652](https://user-images.githubusercontent.com/5098472/110049429-18b16000-7d52-11eb-8120-25066592f9b4.png)
![Screenshot_20210305-012652~2](https://user-images.githubusercontent.com/5098472/110049547-544c2a00-7d52-11eb-8391-57f101c04f44.png)

Cheers
Waog

**Comment by end-ichi** [comment]
We suggested something similar in #615

**Comment by iSoron** [comment]
This suggestion seems related to #342.

**Comment by nikkibecker** [comment]
I think a combo of slightly different color for all 3 states and the number of days in the current streak in place of the x or gray check mark (or maybe all 3) would be great.  Help you see the state of the habit at a glance, and then the number adds psychological motivation to keep the chain going.  I'm much more likely to force myself to do a simple habit when I don't feel like it when I know I'm breaking even a 5 day streak than when I'm simply not checking an 'x'
